src/content/docs/style-guide/structure/styleguide-markup-indentation.mdx

```diff
@@ -0,0 +1,364 @@
+* Item 1
+* Item 2
+```
+
+This is rendered like this:
+
+<img
+  title="Screenshot showing poor formatting of child bullet list"
+  alt="Screenshot showing poor formatting of child bullet list"
+  src={basicListIndentationLeft}
+/>
+
+<Callout variant="tip">
+  You may see some cases in our Markdown where there's two-space indentation for first-level lists. This works on the first level for lists, but it is less semantically accurate.
+</Callout>
+
+## How indent second- and third-level lists [#second-and-third]
+
+For second-level (child) or third-level (grandchild) lists, while you can use three or four spaces to successfully indent child lists, we recommend you use four spaces (or two two-space tabs).
+
+<Callout variant="important">
+  Do not use two-space indentation for any level of list items. While these work in some contexts, they are not predictable. See our [two-space examples](#two-space).
+</Callout>
+
+**Example: Four-space indentation for child (recommended):**
+
+We recommend this since it uses multiples of two spaces:
+
+```
+1. Item 1
+2. Item 2
+    * Item 2a
+    * Item 2b
+```
+
+<img
+  title="Screenshot showing four-space formatting for child"
+  alt="Screenshot showing four-space formatting for child"
+  src={fourSpaceIndentChild}
+/>
+
+**Example: Three-space indentation:**
+
+This works but it's more difficult to type because you have to use a combination of a tab and space to get three total spaces:
+
+```
+1. Item 1
+2. Item 2
+   * Item 2a
+   * Item 2b
+```
+
+<img
+  title="Screenshot showing three-space formatting for child"
+  alt="Screenshot showing three-space formatting for child"
+  src={threeSpaceIndentChild}
+/>
+
+## Problems with two-space indentation for lists [#two-space]
+
+While there are some cases where two-space indentation works, such as indenting images or for parent and child bullet lists, your results in other cases can unpredictable. Here's an example of two-space indentation of a child list. Note that the Markdown library doesn't indent either of these child lists:
+
+**Example: Child bullet list**
+
+```
+1. Item 1
+2. Item 2
+  * Item 2a
+  * Item 2b
+```
+
+Note that it renders the child bullets, but they are not indented:
+
+<img
+  title="Screenshot showing poor formatting of child bullet list"
+  alt="Screenshot showing poor formatting of child bullet list"
+  src={childBulletList}
+/>
+
+**Example: Child numbered list**
+
+```
+1. Item 1
+2. Item 2
+  1. Item 2a
+  2. Item 2b
+```
+Note that the Markdown converter can't tell the indented list is a child:
+
+<img
+  title="Screenshot showing poor formatting of child numbered list"
+  alt="Screenshot showing poor formatting of child numbered list"
+  src={childNumberedList}
+/>
+
+## Bullets nested in numbered list [#bullets-nested-in-numbers]
+
+When you next bulleted lists under number lists, be sure to pay attention to the child indentation. Look at the example below where the initial child bullet list is not fully indented:
```

Comment:
```suggestion
When you nest bulleted lists under number lists, insert four spaces to next the first bullet in the nested list. Look at the example below where the initial child bullet list is not fully indented:
```

Want to specify what SHOULD be done in the first sentence for these headings. That way the reader doesn't have to hunt for the solution within the section, and has it by the first sentence.